BlogPost.md

```diff
@@ -0,0 +1,160 @@
+
+The application can act either as a `node` or a `central` (which are called
+"personalities") depending on the environment variables it has available.
+
+- `Nodes` are geographically spread out doing the actual work of monitoring, for
+  example, local networks.
+- `Central` is a deployment that receives messages from `nodes`, processing and
+  presenting data related to them.
+
+It's also possible to configure the threshold of a `node`, which determines
+their sensitivity in the scanner for threats.
+
+## Deploying the Demo App
+
+### Via a Helm Package file
+
+> _TODO_:  this is a WIP section. Similar to https://docs.starlingx.io/deploy_install_guides/release/kubernetes_access.html#id3
+
+You can simply use the packaged Helm chart from your own source code, and
+use the [Helm CLI](needs-reference) to install the package.
+
+#### Package the Helm chart
+
+```shell
+git checkout https://github.com/bmuniz-daitan/poc-starlingx-messages.git
+cd poc-starlingx-messages
+helm package helm-chart/
+```
+
+The above commands should create a `poc-starlingx-<version>.tgz` file.
+
+```shell
+...
+Successfully packaged chart and saved it to: .../PoC-StarlingX/poc-starlingx-1.5.2.tgz
+```
+
+#### Install the Helm package directly on StarlingX
+
+Once you made the package available on a StarlingX instance, create an
+overrides file (this all should be familiar if you already use Helm packaging)
+. For the demonstration app that we're using, there are many options available
+, but we'll stick to the simplest version of the overrides:
+
+```shell
+cat EOF >> poc-starlingx-messages.yml
+env:
+  - name: MODE
+    value: central
+EOF
+```
+
+Now proceed to installing the Helm package:
+
+```shell
+helm install poc-starlingx-messages -f poc-starlingx-message.yml
+```
+
+#### You're done!
+
+You can see the resources deployed by Helm with the well-known `kubectl`
+commands:
+
+```shell
+kubectl get pods -A
+```
+
+### Via a Helm Repository
+
+> _NOTE_: TODO this is a WIP section
+
+## Application Demo
+
+> _TODO_: Should this section be included in the blog post?
+
+[//]: # (TODO this whole text needs proofreading)
+
+Below you see a `central` running on the right side of the screen and 4
+`node`s running on the left side. `Node`s 1 and 2, on the top, are running with
+a threshold of 5%, which means they are more sensitive, notifying the `central`
+more frequently. `Node`s 3 and 4 at the bottom are running with a threshold of
+20%.
+
+The `central` (to the right) presents the following information:
+
+- a time series of the processed events (based on the timestamp when the event
+  was created on the `node`);
+- a constantly updated list of recently connected `nodes`;
+- a time series of received events (based on `central`'s timestamp when the event
+  was received).
+
+All 4 `node`s, to the left, are constantly generating new data. This random data
+is generated using a long tail distribution and, if above a configured threshold
```

Comment:
I don't know, it may be just me but this information about the type of distribution seems kinda lost in here.

Reply:
Good point, but I'm not touching the Demo just yet. For now I just copied everything from the `Demo.md` and I'm interested in discussing if we want/should keep it or not on the blog post about deployment with Helm.

Reply:
I think that we can explain about the overall work of the app, but the specifics of how the data is generated, I just don't see the relevance here.

Reply:
While I agree with Daniel that this information might not be relevant to the context of **Helm deployment**, I think the app "working" section could be useful in a **blog post** to show the community the capabilities of how StarlingX can handle adversities (like Kubernetes would), when a the poc nodes lose communication with the central but don't lose the information(at least in the past, this feature seemed interesting right?).

**But** if a post only for explaining how the poc works will be created in the feature, then I strongly suggest that this information about the data generated, the tail distribution and related stuff to how the application works, go into the future blogpost **instead** of this one being reviewed.

Reply:
I'll remove the infamous info about the long-tail distribution :laughing: 

@tnpb-encora , the feature of accumulating data when offline is implemented by the app itself and does not make use of any Kubernetes or StarlingX capabilities as of now.